### PR TITLE
Report

### PR DESCRIPTION
**Pull request by mennoo1996**
Wrote parts of the draft report

### CODE REVIEW

File: documentation/final-report/tex/introduction.tex [hunk 1]
```diff
@@ -1 +1,13 @@
-\section{Introduction}
\ No newline at end of file
+\section{Introduction}
+
+During the past 9 weeks, we have worked on designing, implementing and testing a software application. In this report, we will discuss the product and the process that goes with it. We will start by explaining the problem that we tried to solve, and the end-user requirements that we had to take into account.
```
Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: The process of developing it

File: documentation/final-report/tex/introduction.tex [hunk 2]
```diff
@@ -1 +1,13 @@
...
+
+\subsection*{Problem description}
+The software applicaton we have developed was focused on helping PolyCast, a classical recording company with clients world-wide. PolyCast does a lot of classical concert recordings, but this was stil a very manual operation. It involves paper scripts, manual camera operation and a lot of shouting and chaos during a recording. Because of this, camera operators can't be as efficient as they could be when a more automated work flow would be used. This is where we stepped in, and created a software system that focuses on faster, digital script creation that can be linked with a web application, in which camera operators, score callers and directors all have their own view for during live production. We also integrated automatic preset recalling, so that camera operators can focus more on what's happening now, rather than what is coming later.
```
Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: a concert recording company / a classical concert recording company (I believe they do more than classical concerts)

Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: Maybe add a note that we will explain the different roles later?

Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: what is, no abbreviations in a report right?

Comment (mennoo1996) [inline documentation/final-report/tex/introduction.tex]: "Polyhymnia International B.V. is a classical recording company with clients world-wide." -> took that directly from their website

File: documentation/final-report/tex/introduction.tex [hunk 3]
```diff
@@ -1 +1,13 @@
...
+
+\subsection*{End-user requirements}
+The main requirement for the people at PolyCast is that the software application should be easy and fast to use. Especially for the part of the application used during live recordings, there is not a lot of time, so the application should be automated as much as possible, with only a few user interactions needed.\\
```
Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: employees instead of people

Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: with only few user... (-a) ?

File: documentation/final-report/tex/introduction.tex [hunk 4]
```diff
@@ -1 +1,13 @@
...
+For the script-creating part of our application, there were a few requirements set by the people of PolyCast. The timeline orientation should be vertical. Also, it should be easy to create shots, move them around and resize them. Also, they would like it very much if there would be some kind of collision detection so they can see when an "impossible" script was created.
```
Comment (bart-de-jonge) [inline documentation/final-report/tex/introduction.tex]: double also

File: documentation/final-report/tex/introduction.tex (hunk 3 above)
Comment (mennoo1996) [inline documentation/final-report/tex/introduction.tex]: Nope, http://www.usingenglish.com/forum/threads/181765-quot-only-few-quot-or-quot-only-a-few-quot

**Comment by bart-de-jonge** [comment]
Some small comments, other than that great work!

**Comment by Markvanderuit** [comment]
Looks good. I'll merge.

**Pull request merged**